code/modules/mob/living/carbon/brain/posibrain.dm

```diff
@@ -78,17 +78,13 @@ var/global/posibrain_notif_cooldown = 0
 	return
 
 /obj/item/device/mmi/posibrain/proc/transfer_personality(var/mob/candidate)
-	if(brainmob && brainmob.key) //Prevents hostile takeover if two ghosts get the prompt for the same brain.
+	if(brainmob.key) //Prevents hostile takeover if two ghosts get the prompt for the same brain.
 		candidate << "This brain has already been taken! Please try your possesion again later!"
 		return
 	notified = 0
-	brainmob.mind = candidate.mind
```

Comment:
the candidate's mind is now garbage, as being in a posibrain is an irreversible process.
I'd keep this line BUT just before it, have candidate.mind be recreated.
YES this removes antag status and memory, but that was intentional.
The bug here is this line itself, but by recreating the mind it fixes the issue.